gatorgrade/generate/generate.py

```diff
@@ -0,0 +1,16 @@
+"""Generate a YAML file with default messages and specific paths."""
```

Comment:
Instead of "default messages and specific paths", could this docstring reflect the fact that this functionality will generate a GatorGrade configuration file with paths to whitelisted files and default GatorGrader checks?